jekyll/_cci2/migrating-from-jenkins.md

```diff
@@ -83,7 +83,7 @@ Talking about containerization in build systems can be complicated because arbit
 {:.no_toc}
 
 
-If you use a tool like Docker in your workflow, you will likely also want to run it on CI. Jenkins doesn’t provide any built-in support for this, and it is up to you to make sure it is installed and available within your build environment.
+If you use a tool like Docker in your workflow, you will likely also want to run it on CI. Jenkins doesn’t provide any built-in support for this, and it is up to you to make sure it is installed and available within your execution environment.
```

Comment:
```suggestion
If you use a tool like Docker in your workflow, you will likely also want to run it on CI. Jenkins does not provide any built-in support for this, and it is up to you to make sure it is installed and available within your execution environment.
```

Reply:
Should this be "CircleCI" and not "CI"?

Reply:
This is referring generically to continuous integration IIUC